lago_python_client/models/customer_usage.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+from pydantic import BaseModel, Field
+
+class BillableMetric(BaseModel):
+  lago_id: str
+  name: str
+  code: str
+  aggregation_type: str
+
+class Charge(BaseModel):
+  lago_id: str
+  charge_model: str
+
+class ChargeUsage(BaseModel):
+  units: float
+  amount_cents: int
+  amount_currency: str
+  charge: Charge
+  billable_metric: BillableMetric
+
+class CustomerUsageResponse(BaseModel):
+  from_date: str
+  to_date: str
+  issuing_date: str
+  amount_cents: int
+  amount_currency: str
+  total_amount_cents: int
+  total_amount_currency: str
+  vat_amount_cents: int
+  vat_amount_currency: str
```

Comment:
Are you sure that each value in response will always be different than nil?  I had similar issue in my PR. If some value in response is nil, exception will be raised since we didn't define param as optional :)

Reply:
Thanks for pointing this out. I had to look twice, but yes, I confirm all those fields cannot be nil.
They are all generated by the usage service.